README.md

```diff
@@ -46,6 +46,8 @@ pip install .
 Before running for the first time download the related spacy model by running:
 
 ```
+# !python3 -m pip install -U pip setuptools wheel
+# !python3 -m pip install -U spacy
```

Comment:
@eren23 will check

Reply:
Hey, I think we can hold this PR for now as we spoke with @fatihsen20 about it and reached the conclusion of maybe this is a one-off case that happens under certain situations only, therefore I suggest creating issues as soon as we catch other installment problems and once we see a pattern in those we can either update installment instructions or the setup.py file to offer a solution.